templates/UmbracoExtension/.template.config/template.json

```diff
@@ -78,5 +91,20 @@
       "actionId": "210D431B-A78B-4D2F-B762-4ED3E3EA9025",
       "continueOnError": true
     }
```

Comment:
```suggestion
    },
    {
      "actionId": "3A7C4B45-1F5D-4A30-959A-51B88E82B5D2",
      "args": {
        "executable": "powershell",
        "args": "cd Client;npm install;npm run build"
      },
      "manualInstructions": [{
        "text": "From the 'Client' folder run 'npm install' and then 'npm run build'"
      }],
      "continueOnError": true,
      "description ": "Install node modules"
    }
```

Reply:
This was a suggestion for how we could automatically run the npm commands as part of the dotnet template. 
If the user doesn't have powershell it will fail, but will display the manual instructions instead